pyfar/io/io.py

```diff
@@ -297,3 +228,210 @@ def write(filename, compress=False, **objs):
+
+
+    Returns
+    -------
+    signal : Signal
+        :py:class:`~pyfar.classes.audio.Signal` object containing the audio
+        data.
+
+    Notes
+    -----
+    * This function is based on ``soundfile.read``.
+    * Reading int values from a float file will *not* scale the data to
+      [-1.0, 1.0). If the file contains ``np.array([42.6], dtype='float32')``,
+      you will read ``np.array([43], dtype='int32')`` for ``dtype='int32'``.
+    """
+    data, sampling_rate = sf.read(
+        file=filename, dtype=dtype, always_2d=True, **kwargs)
+    signal = Signal(data.T, sampling_rate, domain='time', dtype=dtype)
+    return signal
+
+
+def write_audio(signal, filename, subtype=None, overwrite=True, **kwargs):
+    """
+    Write a :py:class:`~pyfar.classes.audio.Signal` object as a audio file to
+    disk.
+
+    Parameters
+    ----------
+    signal : Signal
+        Object to be written.
+    filename : string, Path
+        Output file. The format is determined from the file extension.
+        See :py:func:`available_formats` for all possible formats.
+    subtype : str, optional
+        The subtype of the sound file, the default value depends on the
+        selected `format` (see :py:func:`default_subtype`).
+        See :py:func:`available_subtypes` for all possible subtypes for
+        a given ``format``.
+    overwrite : bool
+        Select wether to overwrite the audio file, if it already exists.
+        The default is ``True``.
+    **kwargs
+        Other keyword arguments to be passed to ``soundfile.write``.
+
+    Notes
+    -----
+    * Signals are flattened before writing to disk (e.g. a signal with
+      ``cshape = (3, 2)`` will be written to disk as a six channel audio file).
+    * This function is based on ``soundfile.write``.
+    * Amplitudes larger than +/- 1 are clipped.
+
+    """
+    sampling_rate = signal.sampling_rate
+    data = signal.time
+
+    # Reshape to 2D
+    data = data.reshape(-1, data.shape[-1])
+    if len(signal.cshape) != 1:
+        warnings.warn(f"Signal flattened to {data.shape[0]} channels.")
+
+    # Check if file exists and for overwrite
+    if overwrite is False and os.path.isfile(filename):
+        raise FileExistsError(
+            "File already exists,"
+            "use overwrite option to disable error.")
+    else:
+        sf.write(
+            file=filename, data=data.T, samplerate=sampling_rate,
+            subtype=subtype, **kwargs)
+
+
+def read_wav(filename):
+    """
+    Import a WAV file as :py:class:`~pyfar.classes.audio.Signal` object.
+
+    Parameters
+    ----------
+    filename : string, Path
+        Input file.
+
+    Returns
+    -------
+    signal : Signal
+        :py:class:`~pyfar.classes.audio.Signal` object containing the audio
+        data from the WAV file.
+
+    Notes
+    -----
+    * This function is based on :py:func:`read_audio`.
+    """
+    signal = read_audio(filename)
+    return signal
```

Comment:
Should we add a deprecation warning and tests thereof for pyfar 0.5.0 - a assume we want to get rid of this in the future...

Reply:
yes, good point, added that